ccd-definition/AuthorisationCaseField/AuthorisationCaseFieldCaseProg-nonprod.json

```diff
@@ -22,5 +22,21 @@
         "CRUD": "CRU"
       }
     ]
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseFieldID": "featureToggleWA",
```

Comment:
Not sure if required, but systemupdate user should have access to all case data fields - as it is the camunda user / also used by state flow engine for allowing camunda to resolve paths for workflows. May prove useful in future

Reply:
This field will not be updated by Camunda